components/script/dom/headers.rs

```diff
@@ -86,6 +86,7 @@ impl HeadersMethods for Headers {
         if let Some(v) = self.header_list.borrow().get_raw(&valid_name) {
             combined_value = v[0].clone();
             combined_value.push(b","[0]);
+            combined_value.push(b" "[0]);
```

Comment:
@nox @seanmonstar [Here](http://logs.glob.uno/?c=mozilla%23servo&s=26+Jul+2016&e=26+Jul+2016&h=headers#c488830) is a snippet of the irc conversation in regards to append and combine.